modal/functions.py

```diff
@@ -140,7 +141,7 @@ async def _process_result(result, stub, client=None):
         raise RemoteError(result.exception)
 
     try:
-        return deserialize(data, client)
+        return deserialize_data_format(data, data_format, client)
```

Comment:
This is internal to the `_Invocation` class, which I updated to be data format-aware. So FunctionGetOutputs can read ASGI-serialized messages too.